app/models/fe/question_sheet.rb

```diff
@@ -38,7 +38,7 @@ def elements
     end
 
     def all_elements
-      element_ids = pages.pluck(:all_element_ids).compact.join(',').split(',')
+      element_ids = pages.pluck(:all_element_ids).compact.join(',').split(',').find_all(&:present?)
```

Comment:
sorry totally forgot to reply to this.

all_element_ids is a string of comma-separated ids, so something like ["1,2,3","4,5,6"].. so the join makes it "1,2,3,4,5,6", then it needs to get split again to get ["1", "2", ...]

Sometimes I was getting a page with an empty all_elements_ids, so ex. ["1,2", "", "3,4"], which then became "1,2,,3,4" and then split to ["1", "2", "", "3", "4"] and then crashes on the empty id string.

I guess I could also just do

```
element_ids = pages.pluck(:all_element_ids).compact.collect{ |ids| ids.split(',') }.flatten
```

What do you think?